app/manage-your-appointments/2025/10/pharmacy-visits-what-we-learnt-part-1/index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "What we learnt from visiting pharmacies: part 1"
```

Comment:
To help your future selves when you’re trying to find this post, you could make the title a bit more specific rather than "part 1"? Then part 2 could also have a more specific title (about usability issues?)

```suggestion
title: "What we learnt about how pharmacies approach vaccination bookings"
```